core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -461,7 +467,8 @@ class SparkContext(config: SparkConf) extends Logging {
     listenerBus.addToStatusQueue(_statusStore.listener.get)
 
     // Create the Spark execution environment (cache, map output tracker, etc)
-    _env = createSparkEnv(_conf, isLocal, listenerBus)
+    _env = createSparkEnv(_conf, isLocal, listenerBus,
+        new OutputCommitCoordinator(conf, true, Option(this)))
```

Comment:
this looks weird, I think we just need to pass `this` to `createSparkEnv`, which creates driver-side `SparkEnv` that contains `OutputCommitCoordinator`

Reply:
> this looks weird, I think we just need to pass `this` to `createSparkEnv`, which creates driver-side `SparkEnv` that contains `OutputCommitCoordinator`

How about current?

Reply:
Done